app/routes/endemics/species-numbers.js

```diff
@@ -36,7 +41,7 @@ module.exports = [
         return h.view(
           endemicsSpeciesNumbers, {
             backLink,
-            ...getYesNoRadios(legendText.replace('$', speciesEligbileNumberForDisplay), speciesNumbers, session.getEndemicsClaim(request, speciesNumbers), undefined, radioOptions)
+            ...getYesNoRadios(generateLegendText(legendText.replace('$', speciesEligbileNumberForDisplay), claim?.typeOfReview), speciesNumbers, session.getEndemicsClaim(request, speciesNumbers), undefined, radioOptions)
```

Comment:
Can we move the replace $ into the function? Makes it less complicated inline logic and can probably remove the use of .replace()